src/recognizers_store.proto

```diff
@@ -0,0 +1,75 @@
+syntax = "proto3";
+package types;
+
+// PatternRecognizer is the object stored in the store
+message PatternRecognizer {
+    // The name of the recognizer, unique
+    string name = 1;
+    // The Regex used by this recognizer
+    string regex = 2;
```

Comment:
The pattern is a message on its own. Its structure is: 
message Pattern:
string name = 1
string regex = 2
float strength = 3
A PatternRecognizer should have a list of such patterns